HISTORY.rst

```diff
@@ -6,7 +6,7 @@ History
 Pending
 -------
 
-* New release notes here
+* Use pip-compile and requirements.in
```

Comment:
This doesn't warrant a history note because it doesn't affect users of the library